notify-github.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# MOVE THIS FILE TO  https://github.com/libero/scripts/ once tested
+set -e
+
+unique_id=$(uuidgen)
+temp_file="github-commit-status-${unique_id}.log"
+
+repository="libero/reviewer"
+# Travis CI builds a merge commit on pull requests, so
+# we want to refer to the second parent of this merge commit
+commit=$(git rev-parse HEAD^2)
```

Comment:
this bit makes it specific for PRs, but we don't necessarily need to do this notification for `master` commits. If we do, links could be seen on the various :heavy_check_mark: symbols at https://github.com/libero/reviewer/commits/master

Reply:
if we want to make this usable on `master`, extract this command and pass in `commit` as a variable